src/canmatrix/canmatrix.py

```diff
@@ -551,6 +552,48 @@ def pack_bitstring(length, is_float, value, signed):
 
     return bitstring
 
+@attr.s(cmp=False)
+class Arbitration_Id(object):
+    standard_id_mask = ((1 << 11) - 1)
+    extended_id_mask = ((1 << 29) - 1)
+    compound_extended_mask = (1 << 31)
+
+    id = attr.ib(default=None)
+    extended = attr.ib(default=None)
+
+    def __attrs_post_init__(self):
```

Comment:
I'm tending away from writing `__attrs_post_init__`s anymore.  I think this could be just a helper `@classmethod` that the `from_compound_integer()` calls.  We should be _able_ to construct any class we want rather than having to create it and then modify to get to any arbitrary set of attributes.